version.m4

```diff
@@ -5,6 +5,8 @@ define([PRODUCT_VERSION_RESOURCE], [9,0,0,21])
 define([PRODUCT_TAP_WIN_COMPONENT_ID], [tap0901])
 define([PRODUCT_TAP_WIN_MAJOR], [9])
 define([PRODUCT_TAP_WIN_MINOR], [21])
+define([PRODUCT_TAP_WIN_REVISION], [2])
```

Comment:
This variable is not used for anything right now, but I think it should be here for clarity.